mapie/tests/test_classification.py

```diff
@@ -267,30 +278,31 @@
 }
 
 COVERAGES = {
-    "score": 6 / 9,
-    "score_cv_mean": 1,
-    "score_cv_crossval": 1,
-    "cumulated_score_include": 1,
-    "cumulated_score_not_include": 5 / 9,
-    "cumulated_score_randomized": 6 / 9,
-    "cumulated_score_include_cv_mean": 1,
-    "cumulated_score_not_include_cv_mean": 5 / 9,
-    "cumulated_score_randomized_cv_mean": 5 / 9,
-    "cumulated_score_include_cv_crossval": 2 / 9,
-    "cumulated_score_not_include_cv_crossval": 0,
-    "cumulated_score_randomized_cv_crossval": 4 / 9,
-    "naive": 5 / 9,
-    "top_k": 1,
-    "raps": 1,
-    "raps_randomized": 8/9
+    'score': 0.6666666666666666,
```

Comment:
Why changing the format ? Moreover, to make the review easier please change only what is needed to be changes (typically, the results for non cross-validation methods should not be changed)

Reply:
Because I have changed the parameter `shuffle=True` in `check_cv` function (see in `utils.py`), all the results are impacted. In fact, I do copy/paste. I agree to changing the format.